src/Analyzers/CSharp/Tests/AddBraces/AddBracesTests.cs

```diff
@@ -665,7 +659,7 @@ static void Main()
         /// <summary>
         /// Verifies that the use of braces in a construct nested within the true portion of an <c>if</c> statement does
         /// not trigger the multiline behavior the <c>else</c> clause of a containing stetemnet for
-        /// <see cref="PreferBracesPreference.WhenMultiline"/>. The <c>else</c> clause would only need braces if the true
+        /// "PreferBracesPreference.WhenMultiline". The <c>else</c> clause would only need braces if the true
```

Comment:
❔ Can we not keep this reference?

Reply:
![DocCommentIssue](https://user-images.githubusercontent.com/10605811/77021342-aa5a6e80-6943-11ea-9ff3-84630809b54b.png)

I was perplexed too - seems like a doc comment compiler bug.